common/output-model.cpp

```diff
@@ -11,6 +11,7 @@
 #include <librealsense2/hpp/rs_internal.hpp>
 
 using namespace rs2;
+using namespace utilities::strings;
```

Comment:
We should avoid `using namespace...`, instead naming classes that we're using: `using utilities::string::...`